modules/videoio/src/ffmpeg_codecs.hpp

```diff
@@ -78,6 +78,7 @@ typedef struct AVCodecTag {
 } AVCodecTag;
 
 #if (LIBAVCODEC_VERSION_INT <= AV_VERSION_INT(54, 51, 100))
+#define AV_CODEC_ID_HEVC CODEC_ID_HEVC
```

Comment:
Perhaps `AV_CODEC_ID_HEVC` is not defined for all mentioned ffmpeg versions. Some distributions like CentOS likes to use very old versions of FFmpeg API.

We need proper check here and in other places.

For example, we could reuse `AV_CODEC_ID_H265` [definition](https://github.com/FFmpeg/FFmpeg/blob/release/3.4/libavcodec/avcodec.h#L395):
```
#ifdef AV_CODEC_ID_H265
... use AV_CODEC_ID_H265 ...
#endif
```